src/functionaltest/java/com/ericsson/ei/frontend/SubscriptionHandlingFunctionality.java

```diff
@@ -45,20 +50,68 @@ public void testSubscription() throws Exception {
         assert (new WebDriverWait(driver, 10)
                 .until((webdriver) -> subscriptionPage.presenceOfHeader(subscriptionHeaderID)));
 
-        // // Press "Reload" button and verify that two subscriptions with names
-        // "Subscription1" and "Subscription2" are present
+        // Press "Reload" button without enabling LDAP and verify that two subscriptions
+        // with names
+        // "Subscription1" and "Subscription2" are present AND there exists "edit" and
+        // "delete buttons" for unauthorized user "ABCD"
         String response = this.getJSONStringFromFile(RELOAD_TEST_FILE_PATH);
+        String deletePath = "//tr[td[contains(.,'Subscription1')]]/td/button[contains(text(),'Delete')]";
```

Comment:
I would really like a bit more explainable name, like deleteButtonXPath. Got confused later on what this actually represented. Same for the other paths